### DISCUSSION TITLE
Unable to select templates for "Send booking related email to selected bookings"

### DISCUSSION THREAD

**Discussion by refugeindhamma** [liedekef/events-made-easy#217] [Q&A] [answered]
Hi Franky,

I'm unable to select templates for both Subject & Body for "Send booking related email to selected bookings" in "Approved Bookings:".
![Screenshot 2023-07-01 115718](https://github.com/liedekef/events-made-easy/assets/101639308/01897cba-0571-464f-86f6-10b5de806f6c)

I would like to use these 2 templates to re-send the reminders to people who can't find/deleted their reminder emails:
![image](https://github.com/liedekef/events-made-easy/assets/101639308/8f9f6714-3ff8-42c0-a83e-487f8e8aff7e)

Edit: I'm also unable to select templates for PDF and HTML outputs.

**Comment by liedekef** [comment, marked as answer]
That is because your templates are of type "All". I mostly try to be strict in this, so only show the correct templates where possible. But for mails I sometimes do it not consistently, so this will fix it for you: https://github.com/liedekef/events-made-easy/commit/21ef475116d9d6d32b9ceb936b0789e2fcb9279f
Best is to change the type "All" to "RSVP related mail" or so.

For PDF and HTML: there I only show templates that are actually of type PDF or HTML, nothing else. PDF needs more info when creating the template, and HTML might need more layout, so it is logical.
